packages/sdk/package.json

```diff
@@ -2,21 +2,36 @@
   "name": "@ordzaar/ordit-sdk",
   "version": "0.0.0",
   "description": "SDK for simplified interaction with wallet APIs and inscription service",
-  "main": "./dist/index.js",
-  "types": "./dist/index.d.ts",
+  "type": "module",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.js",
+  "types": "./index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    },
+    "./browser-wallets": {
+      "import": "./dist/browser-wallets.js",
+      "require": "./dist/browser-wallets.cjs"
+    }
+  },
   "files": [
-    "dist"
+    "dist/*",
+    "index.d.ts",
+    "browser-wallets.d.ts"
   ],
   "keywords": [
     "ordit",
     "ordit-sdk",
     "sdk"
   ],
   "scripts": {
-    "dev": "vite",
-    "build": "tsc && vite build",
-    "build:watch": "tsc && vite build --watch",
-    "preview": "vite preview"
+    "dev": "pnpm clean && tsc && vite build --watch",
+    "build": "pnpm clean && tsc && vite build",
+    "build:watch": "pnpm clean && tsc && vite build --watch",
+    "preview": "vite preview",
+    "clean": "rm -rf dist"
```

Comment:
also `rm -rf` might not work for windows haha

Reply:
kevin watching out for poor windows users

Reply:
I use windows as well FYI :D